crates/loader/src/local.rs

```diff
@@ -251,6 +259,54 @@ impl LocalLoader {
         file_content_ref(path)
     }
 
+    async fn load_registry_source(
+        &self,
+        registry: Option<impl AsRef<str>>,
```

Comment:
nit: If this isn't a public API, I don't think we need to pay the compilation price for making this generic.

```suggestion
        registry: Option<&str>,
```